saas/api/roles.py

```diff
@@ -794,6 +802,14 @@ class RoleDetailAPIView(RoleMixin, DestroyAPIView):
         DELETE /api/profile/cowork/roles/managers/xia/ HTTP/1.1
     """
 
+    def post(self, request, *args, **kwargs):
+        # TODO add docs
```

Comment:
1/ I believe an update to the parent class is missing (before `def post` it was `DestroyAPIView`). Update to `RetrieveUpdateDestroyAPIView`, and implement `def retrieve` as a result.
2/ Most likely the best is to return the `Role` instead of `NO CONTENT` such that the UI can display a nice message "The invite was sent to _Full name &lt;email&gt;_".

Documentation:

        """
        Re-sends the invite e-mail that the user was granted a role on the organization.

        **Examples

        .. code-block:: http

            POST /api/profile/cowork/roles/manager/xia/ HTTP/1.1

       responds

        .. code-block:: json

                            {
                    "created_at": "2018-01-01T00:00:00Z",
                    "role_description": {
                        "name": "Manager",
                        "slug": "manager",
                        "organization": {
                            "slug": "cowork",
                            "full_name": "ABC Corp.",
                            "printable_name": "ABC Corp.",
                            "created_at": "2018-01-01T00:00:00Z",
                            "email": "support@localhost.localdomain"
                        }
                    },
                    "user": {
                        "slug": "alice",
                        "email": "alice@localhost.localdomain",
                        "full_name": "Alice Doe",
                        "created_at": "2018-01-01T00:00:00Z"
                    },
                    "request_key": "1",
                    "grant_key": null
                }
    """

Reply:
> I believe an update to the parent class is missing (before def post it was DestroyAPIView).

Which update to which class?